subprojects/platform-base/build.gradle.kts

```diff
@@ -37,6 +37,10 @@ dependencies {
     integTestDistributionRuntimeOnly(project(":distributions-core"))
 }
 
+strictCompile {
+    ignoreDeprecations() // uses deprecated software model types
```

Comment:
This will hide any usage of deprecated members in the whole project. I would have added warning suppression annotation in places where deprecated members are in use instead.

Reply:
Good point, I will fix this for the whole set of PRs.